src/main/java/teammates/logic/api/Logic.java

```diff
@@ -1346,4 +1346,7 @@ public boolean isStudentsInSameTeam(String courseId, String student1Email, Strin
         return studentsLogic.isStudentsInSameTeam(courseId, student1Email, student2Email);
     }
 
+    public void updateToEnsureValidityOfInstructorsForTheCourse(String courseId, InstructorAttributes instructorToEdit) {
```

Comment:
Take a look at other methods in this `Logic` class. There's something done in all of them that is not done here.